specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-08-01/workbooks_API.json

```diff
@@ -0,0 +1,808 @@
+        }
+      },
+      "type": "object",
+      "description": "The parameters that can be provided when updating workbook properties properties."
+    },
+    "WorkbookPropertiesUpdateParameters": {
+      "description": "Properties that contain a workbook for PATCH operation.",
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "The user-defined name (display name) of the workbook."
+        },
+        "serializedData": {
+          "type": "string",
+          "description": "Configuration of this particular workbook. Configuration data is a string containing valid JSON"
+        },
+        "category": {
+          "type": "string",
+          "description": "Workbook category, as defined by the user at creation time."
+        },
+        "tags": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "A list of 0 or more tags that are associated with this workbook definition"
+        },
+        "description": {
+          "type": "string",
+          "x-nullable": true,
+          "description": "The description of the workbook."
+        },
+        "revision": {
+          "type": "string",
+          "x-nullable": true,
+          "description": "The unique revision id for this workbook definition"
+        }
+      }
+    },
+    "WorkbookError": {
+      "description": "Error response.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorDefinition",
+          "description": "The error details."
+        }
+      }
+    },
+    "ErrorDefinition": {
+      "description": "Error definition.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Service specific error code which serves as the substatus for the HTTP error code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "Description of the error.",
+          "type": "string",
+          "readOnly": true
+        },
+        "innererror": {
+          "description": "Internal error details.",
+          "items": {
+            "$ref": "#/definitions/InnerErrorTrace"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "InnerErrorTrace": {
+      "description": "Error details",
+      "type": "object",
+      "properties": {
+        "trace": {
+          "description": "detailed error trace",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "WorkbookManagedIdentity": {
+      "description": "Customer Managed Identity",
+      "type": "object",
+      "properties": {
+        "userAssignedIdentities": {
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/WorkbookUserAssignedIdentities"
```

Comment:
>WorkbookUserAssignedIdentities [](http://example.com/codeflow?start=35&length=30)

If possible please refactor the $ref to UserAssignedIdentity definition in common managed identity type definitions 

[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/managedidentity.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/managedidentity.json)